ai-platform-unified/notebooks/official/automl/tabular-model-training-and-prediction.ipynb

```diff
@@ -0,0 +1,733 @@
+{
```

Comment:
<p>Can you change it to install <code>google-cloud-aiplatform</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>